lib/include/jxl/decode.h

```diff
@@ -253,17 +253,22 @@ typedef enum {
  */
 JXL_EXPORT void JxlDecoderRewind(JxlDecoder* dec);
 
-/** Makes the decoder skip the next `amount` frames. If this is used after
- * JxlDecoderRewind to skip frames that were already decoded before, the decoder
- * is able to skip more efficiently, as it can skip decoding earlier frames that
- * it knows later frames don't depend on. A frame here is defined as a frame
- * that without skipping emits events such as JXL_DEC_FRAME and JXL_FULL_IMAGE,
- * frames that are internal to the file format but are not rendered as part of
- * an animation, or are not the final still frame of a still image, are not
- * counted. If the decoder is already processing a frame (could have emitted
- * JXL_DEC_FRAME but not yet JXL_DEC_FULL_IMAGE), it starts skipping from the
- * next frame. If the amount is larger than the amount of frames remaining in
- * the image, all remaining frames are skipped.
+/** Makes the decoder skip the next `amount` frames. It still needs to process
+ * the input, but will not output the frame events and can be more efficient. If
+ * this is used after JxlDecoderRewind to skip frames that were already decoded
+ * before, the decoder is able to skip more efficiently since it has more
+ * information about which frames reference which. A frame here is defined as a
+ * frame that without skipping emits events such as JXL_DEC_FRAME and
+ * JXL_FULL_IMAGE, frames that are internal to the file format but are not
+ * rendered as part of an animation, or are not the final still frame of a still
+ * image, are not counted. If the decoder is already processing a frame (could
+ * have emitted JXL_DEC_FRAME but not yet JXL_DEC_FULL_IMAGE), it starts
+ * skipping from the next frame. If the amount is larger than the amount of
+ * frames remaining in the image, all remaining frames are skipped. If frames
+ * are already being skipped when this function is called, it is ignored and
```

Comment:
Is this line correct? It seems to clash with the comment in the impl

Reply:
That was not supposed to be there, fixed, PTAL